src/audio/src/CMakeLists.txt

```diff
@@ -1,9 +1,31 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof src_generic.c src_hifi2ep.c src_hifi3.c src_hifi4.c src_hifi5.c src_common.c src.c)
+set(base_files src_generic.c src_hifi2ep.c src_hifi3.c src_hifi4.c src_hifi5.c src_common.c src.c)
 
 if(CONFIG_IPC_MAJOR_3)
-	add_local_sources(sof src_ipc3.c)
+  list(APPEND base_files src_ipc3.c)
 elseif(CONFIG_IPC_MAJOR_4)
-	add_local_sources(sof src_ipc4.c)
+  list(APPEND base_files src_ipc4.c)
+endif()
+
+sof_list_append_ifdef(CONFIG_COMP_SRC_LITE src_lite.c)
+
+is_zephyr(it_is)
+if(it_is) ###  Zephyr ###
```

Comment:
Would it not be simpler and more readable with
```
if(is_zephyr)
```

Reply:
@lgirdwood there's a slight risk of broken telephone when I've been continueing Marc's work, but this would seem a standard cmake macro trick. possible more readable:
```
-is_zephyr(it_is)
-if(it_is) ###  Zephyr ###
+is_zephyr(zephyr)
+if(zephyr) ###  Zephyr ###
```

...?

Reply:
ok, lest factor this improvement in as we go, not blocking this one though.